.github/workflows/ci.yml

```diff
@@ -12,5 +12,21 @@ permissions:
   contents: read
 
 jobs:
-  Shared:
-    uses: fog/.github/.github/workflows/ci.yml@v1.4.0
```

Comment:
We'll want to drop this and use the shared one again I think once we are satisfied (though perhaps we'll want to update the shared one with RUBYOPTS in some way also).

Reply:
@geemus I just had the idea, but I can create a shared config on `fog/.github` for `fog-aws` `fog-xml` and `fog-core`. 

and once all gems have migrated to this new shared config. we can completely remove the old config.

If you agree with this, please let me know and I'll go ahead and create a PR on `fog/.github` :ok_hand:

Reply:
Also or instead, we could leave this for now, but also add the shared workflow back (so that it just runs both). I didn't want to lose the shared workflow. Both is probably redundant, but it would be ok as an interim step.